Makefile

```diff
@@ -32,5 +32,8 @@ generate: oc build
 	mkdir -p $(DEST_DIR)/communication-matrix
 	./$(EXECUTABLE) -format=$(FORMAT) -env=$(CLUSTER_ENV) -destDir=$(DEST_DIR)/communication-matrix -deployment=$(DEPLOYMENT) -customEntriesPath=$(CUSTOM_ENTRIES_PATH) -customEntriesFormat=$(CUSTOM_ENTRIES_FORMAT)
 
+apply-firewall: build
+	./$(EXECUTABLE) -applyFirewall -format=$(FORMAT) -env=$(CLUSTER_ENV) -destDir=$(DEST_DIR)/communication-matrix -deployment=$(DEPLOYMENT)
```

Comment:
format and destDir are not used

Reply:
sorry destDir is used but I'm wondering if apply command should just be apply and allow to generate the mat in nftables format

Reply:
thats whats doing its applying the firewall by that mat and generate the nftformat